server/@types/dto/index.d.ts

```diff
@@ -0,0 +1,217 @@
+declare module 'dto' {
+  /**
+   * Interface defining common reference and audit related properties that DTO types can inherit through extension.
+   */
+  interface ReferencedAndAuditable {
+    reference: string
+    createdBy: string
+    createdByDisplayName: string
+    createdAt: Date
+    createdAtPrison: string
+    updatedBy: string
+    updatedByDisplayName: string
+    updatedAt: Date
+    updatedAtPrison: string
+  }
+
+  export interface InductionDto extends ReferencedAndAuditable {
```

Comment:
Everything in from here is a copy and paste from the API definitions; replacing the term "Response" with "Dto", and changing the date fields to `Date`s (in the above interface)